src/constants/constants.js

```diff
@@ -11,7 +11,7 @@ export const PACKAGE_PATH = {
 }
 
 // BASE PATHS
-export const BASE_PATH = 'app/design/frontend/Snowdog/'
+export const BASE_PATH = 'app/design/frontend/'
```

Comment:
base path meaning isn't obvious to me. what do you think about including theme in the name?

Reply:
ok